.github/workflows/trivy.yaml

```diff
@@ -8,20 +8,28 @@ jobs:
   trivy_scan:
     if: github.event.issue.pull_request && github.event.comment.body == '/trivy'
     runs-on: ubuntu-latest
-    permissions:
```

Comment:
You can keep it as `permissions: {}` to safely disable all the possible permissions -> https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token#defining-access-for-the-github_token-permissions.

Reply:
We still need read permissions for pull requests, otherwise you can't checkout the PR to build it.